stories/index.js

```diff
@@ -41,7 +41,26 @@ storiesOf('Picker', module)
       showPreview={boolean('Show preview', true)}
       showSkinTones={boolean('Show skin tones', true)}
       custom={CUSTOM_EMOJIS}
-    />
+      notFoundImg={text('Not found image', 'sleuth_or_spy')}
```

Comment:
As said above, you should not need to specify this; the fact that you do means we're breaking backwards compatibility.